react/react1/week1/todo-list-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+
+import React from "react";
+import "./App.css";
+import todoList from "./todoList.json";
+import TodoListTitle from "./todoListTitle";
+import TodolistItem from "./todoListItem";
+
+
+function App() {
+  return (
+    <div className="App">
+      {<TodoListTitle/>}
```

Comment:
Hi Sergii
nice job :-) thank you. 
you don't need to use { } "curly brackets" to render the component.